package.json

```diff
@@ -4,7 +4,7 @@
   "description": "GitHub App that syncs upstream repositories with forks using PRs",
   "scripts": {
     "lint": "eslint src/*",
-    "test": "jest"
+    "test": "jest --runInBand"
```

Comment:
Note that this is required since multiple test suites (a file is a suite) now mock `fs` and so they can't run in parallel.